fv3gfs-util/requirements.txt

```diff
@@ -15,7 +15,7 @@ black==19.10b0
 numcodecs==0.7.2 #pin for gt4py, h5py and py3.6 to agree
 h5py==2.10.0 #pin for gt4py, h5py and py3.6 to agree
 h5netcdf
-dask==2021.03.0 #py3.6 - last supported version
+dask>=2021.10.0
```

Comment:
You need to also update the constraints at the top level

Reply:
Updated.